src/cryptography/x509/base.py

```diff
@@ -652,14 +675,14 @@ def not_valid_after(self, time):
             self._extensions,
         )
 
-    def add_extension(self, extension, critical):
+    def add_extension(self, extval: ExtensionType, critical: bool):
```

Comment:
If we're going to change the name of this arg it needs to be updated in the docs (and this would break kwarg users right?)

Reply:
It's hard to know anything for certain, but I claim that there's no reason you'd ever pass this value as a kwarg. I'll go fix the docs.